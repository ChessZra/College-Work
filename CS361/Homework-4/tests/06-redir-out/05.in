file file
echo Hello >file;echo World >file;echo ! >file
file file
md5sum file
cat file
rm file

