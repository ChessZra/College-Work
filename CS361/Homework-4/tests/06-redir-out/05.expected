cs361sh> file: cannot open `file' (No such file or directory)
cs361sh> cs361sh> file: ASCII text
cs361sh> 8183aa57a23658efe7ba7aebe60816bc  file
cs361sh> !
cs361sh> cs361sh> cs361sh> 
